### ISSUE TITLE
[REQUEST] Remote location for the chromium brotli file

### ISSUE THREAD

**Issue by Sparticuz** [Sparticuz/chromium#18]
Implement the ability to have the chromium brotli file stored somewhere besides the package. This would enable us to get around 50mb limits that seem to plague vercel and other services that limit the deployable size.
```
test("Check the page title of example.com", async (t) => {
  const browser = await puppeteer.launch({
    args: chromium.args,
    defaultViewport: chromium.defaultViewport,
    executablePath: await chromium.executablePath("https://bucket.s3.amazonaws.com/chromium108.br"),
    headless: chromium.headless,
    ignoreHTTPSErrors: true,
  });
});

test("Check the page title of example.com", async (t) => {
  const browser = await puppeteer.launch({
    args: chromium.args,
    defaultViewport: chromium.defaultViewport,
    executablePath: await chromium.executablePath("/opt/chromium.br"),
    headless: chromium.headless,
    ignoreHTTPSErrors: true,
  });
});
```

Closes #24 
Closes #8

**Comment by Sparticuz** [comment]
This was only partially fixed in #26. I still would like to implement the remote brotli option.

**Comment by Sparticuz** [comment]
v109.0.6 has been published with now lets you use a remote location for the brotli files (now a .tar pack). Super thanks to @aqaurius6666 for the code!!
